packages/react-components/src/components/chart/types.ts

```diff
@@ -93,17 +93,24 @@ export type ChartOptions = {
   fontSettings?: SimpleFontSettings;
   legend?: ChartLegend;
   significantDigits?: number;
-  graphic?: InternalGraphicComponentGroupOption[]; // This needs to be removed from the public api
+  graphic?: InternalGraphicComponentGroupOption[];
   theme?: string;
   groupId?: string;
   id?: string;
 };
 
+export interface ViewportInMs {
+  end: number;
+  initial: number;
+  widthInMs: number;
```

Comment:
Can we move something like this to a util function? I feel like width in milliseconds doesn't really make sense? Assuming this is useful in trendcursors for knowing where on the xaxis to place something?

Reply:
the invoking function need not use this, I believe end and initial are probably more used and this is available if needed .. its a simple math, so dont think it will hog up too much processing cycle anyway..